gymnasium/spaces/space.py

```diff
@@ -1,26 +1,19 @@
 """Implementation of the `Space` metaclass."""
+from __future__ import annotations
 
-from typing import (
-    Any,
-    Generic,
-    Iterable,
-    List,
-    Mapping,
-    Optional,
-    Sequence,
-    Tuple,
-    Type,
-    TypeVar,
-    Union,
-)
+from typing import Any, Generic, Iterable, Mapping, Sequence, TypeVar
 
 import numpy as np
+import numpy.typing as npt
 
 from gymnasium.utils import seeding
 
 T_cov = TypeVar("T_cov", covariant=True)
 
 
+MASK_NDARRAY = npt.NDArray[np.int8]
```

Comment:
I don't think it should be in all caps, it's a type synonym and not really a constant (so `MaskNDArray` or something)


Also - please double-check that it should indeed always be a `np.int8` ndarray, I haven't really kept up with the sample mask feature

Reply:
Updated